lib/active_fedora/solr_query_builder.rb

```diff
@@ -44,8 +44,14 @@ def construct_query_for_rel(field_pairs, join_with = ' AND ')
       # @example
       #   construct_query([['library_id_ssim', '123'], ['owner_ssim', 'Fred']])
       #   # => "_query_:\"{!raw f=library_id_ssim}123\" AND _query_:\"{!raw f=owner_ssim}Fred\""
-      def construct_query(field_pairs, join_with = ' AND ')
-        pairs_to_clauses(field_pairs).join(join_with)
+      def construct_query(field_pairs, join_with = default_join_with)
+        clauses = pairs_to_clauses(field_pairs)
+        return clauses.join("") if clauses.count <= 1
```

Comment:
would `clauses.first` be more efficient? Then we don't have to allocate a new string.